web/thesauruses/clojure/1/operators.json

```diff
@@ -0,0 +1,213 @@
+    },
+    "greater_than": {
+      "code": "(> 8 1)",
+      "name": "Greater than operator"
+    },
+    "greater_than_or_equal_to": {
+      "comment": "All Clojure comparison functions can take arbitrarily many arguments.",
+      "code": "(>= 12 10 8)",
+      "name": "Greater than or equal to operator"
+    },
+    "null_coalescing": {
+      "not-implemented": "true",
+      "name": "Null coalescing operator"
+    },
+    "is": {
+      "not-implemented": "true",
+      "name": "Is operator"
+    },
+    "is_not": {
+      "not-implemented": "true",
+      "name": "Is not operator"
+    },
+    "logical_and": {
+      "comment": "As with most operator functions, arbitrarily many arguments are supported",
+      "code": "(and (> 10 2) (= 3 (/ 33 11) cond1)",
+      "name": "Logical AND Operator"
+    },
+    "logical_or": {
+      "comment": "As with most operator functions, arbitrarily many arguments are supported. For `or`, the first truthy clause is returned as the result of the overall expression.",
+      "code": "(or x y z)",
+      "name": "Logical OR Operator"
+    },
+    "logical_not": {
+      "code": "(not (< x 15))",
+      "name": "Logical NOT Operator"
+    },
+    "bitwise_and": {
+      "code": "(bit-and 2r1100 2r1001)",
+      "name": "Bitwise AND operator"
+    },
+    "bitwise_and_assignment": {
+      "not-implemented": "true",
+      "name": "Bitwise AND and assignment operator"
+    },
+    "bitwise_or": {
+      "code": "(bit-or 2r1100 2r1001)",
+      "name": "Bitwise OR operator"
+    },
+    "bitwise_or_assignment": {
+      "not-implemented": "true",
+      "name": "Bitwise OR and assignment operator"
+    },
+    "bitwise_not": {
+      "code": "(bit-not 2r0111)",
+      "name": "Bitwise NOT operator"
+    },
+    "not_assignment": {
+      "not-implemented": "true",
+      "name": "Bitwise NOT and assignment operator"
+    },
+    "bitwise_xor": {
+      "code": "(bit-xor 2r1100 2r1001) ",
+      "name": "Bitwise XOR operator"
+    },
+    "bitwise_xor_assignment": {
+      "not-implemented": "true",
+      "name": "Bitwise XOR and assignment operator"
+    },
+    "bitwise_xnor": {
+      "not-implemented": "true",
+      "name": "Bitwise XNOR operator"
+    },
+    "bitwise_xnor_assignment": {
+      "not-implemented": "true",
+      "name": "Bitwise XNOR and assignment operator"
+    },
+    "left_shift": {
+      "code": "(bit-shift-left 1 10)",
+      "name": "Left shift bitwise operator"
+    },
+    "left_shift_assignment": {
+      "not-implemented": "true",
+      "name": "Left shift assignment operator"
+    },
+    "right_shift": {
+      "code": "(bit-shift-right 100 2)",
+      "name": "Right shift bitwise operator"
+    },
+    "right_shift_assignment": {
+      "not-implemented": "true",
+      "name": "Right shift assignment operator"
+    },
+    "ternary": {
+      "comment": "Use the `if` function to provide a ternary operator equivalent.",
+      "code": "(if (> 2 1) \"Yes\" \"No\")",
```

Comment:
This is also inventing a oslution. Should be `not-implemented` too.

Reply:
I actually think in this case it truly is equivalent.  Since every expression returns a value, this is how you would accomplish the same thing that requires the operator in other languages.  See [here](https://reshmeeauckloo.wordpress.com/2015/12/13/clojure-if-cond-case/) for a better explanation than I can give.

Nonetheless I don't feel that strongly about it, so I marked not-implemented.

Reply:
I think I also misread it because when I look at it now, it clearly works like a ternary. I can either merge this in or you can roll this back.